libs/s25main/world/MapBase.cpp

```diff
@@ -182,3 +182,9 @@ MapPoint MapBase::MakeMapPoint(Position pt) const
 {
     return ::MakeMapPoint(pt, size_);
 }
+
+unsigned MapBase::numPointsRadius(unsigned radius, bool includePt) const {
+    // For every additional radius we get 6 * curRadius more points. Hence we have 6 * sum(1..radius) points + the
+    // center point if requested This can be reduced via the gauss formula to the following:
+    return (radius * radius + radius) * 3u + (includePt ? 1u : 0u);
+}
```

Comment:
Watch out for this: Github puts a symbol here to show "No newline at end of file".

So make sure each file ends with an empty line. Otherwise if you add e.g. a new function here (best example: what you did here) it would show this line as changed (added newline at the end of this line) instead of nicely adding (here 6) new lines (as it does now because the file had an empty line at the end)